### ISSUE TITLE
Play remote files

### ISSUE THREAD

**Issue by realrecordzLab** [jaredpetersen/troubadour#31]
Is possible to play a remote file using this library?

**Comment by jaredpetersen** [comment]
Nope. This is just a [wrapper over various CLI tools](https://github.com/jaredpetersen/troubadour/blob/master/lib/supportedAudioPlayers.json) for playing audio and I believe they all require the file to be on disk.

**Comment by realrecordzLab** [comment]
> Nope. This is just a [wrapper over various CLI tools](https://github.com/jaredpetersen/troubadour/blob/master/lib/supportedAudioPlayers.json) for playing audio and I believe they all require the file to be on disk.

with `spawn('path/to/vlc' ['https://foo.bar'])` vlc is able to play a remote file from url. It will be a nice feature to have in this library

**Comment by jaredpetersen** [comment]
Yeah, it also looks like there is [some checking done](https://github.com/jaredpetersen/troubadour/blob/de8e6c811cadb9920c1c4bd94912089c74947ae2/index.js#L36-L44) on the provided filepath to ensure that it's on disk. We support VLC via `cvlc` but the filepath checks will still ruin your day.

**Comment by realrecordzLab** [comment]
> Yeah, it also looks like there is [some checking done](https://github.com/jaredpetersen/troubadour/blob/de8e6c811cadb9920c1c4bd94912089c74947ae2/index.js#L36-L44) on the provided filepath to ensure that it's on disk. We support VLC via `cvlc` but the filepath checks will still ruin your day.

I saw it and tried to remove but library stop working
